Scripts/Runtime/Data/Collections/UnsafeTypedStream.cs

```diff
@@ -112,15 +114,36 @@ public int ElementsPerBlock
         {
             get => m_BufferInfo->BlockSize / ELEMENT_SIZE;
         }
+        
+        /// <summary>
+        /// Convenience constructor that fits as many elements into a 16kb block size and allows you to specify the
+        /// number of lanes.
```

Comment:
```suggestion
        /// Convenience constructor that fits as many elements into a 16kb block size as possible and allows you
        /// to specify the number of lanes.
```

Reply:
This typo appears in all the constructor docs.

Reply:
Fixed.